siac/parse.go

```
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+// parseSize converts strings of form 10GB to a size in bytes
+// If parsing fails the original value is returned unmodified
```

parseSize should return an error if parsing fails